signalfx/resource_signalfx_single_value_chart.go

```diff
@@ -219,13 +225,16 @@ func getSingleValueChartOptions(d *schema.ResourceData) *chart.Options {
 	}
 
 	var programOptions *chart.GeneralOptions
+	if programOptions == nil {
```

Comment:
This isn't doing anything since programOptions is always nil one line after the variable is declared. I think the intention was to have programOptions be nil unless some value in it was set.

So I think the correct thing to do is in the `if val, ok := d.GetOk("timezone"); ok {` block add

```go
if programOptions == nil {	
    programOptions = &chart.GeneralOptions{}	
}
```

Reply:
Sorry for making silly mistake...
It was a copy-paste error.
fixed it now.